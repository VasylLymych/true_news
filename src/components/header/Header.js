import React from "react";
import logo from './../../logo.jpg'
import styles from './Header.module.css'

const Header = () => {
    return (
        <header className={styles.header}>
            <div className={styles.logo_wrapper}>
                <img src={logo} className={styles.logo_img} alt="logo"/>
                <h1>Новини України та світу</h1>
            </div>
        </header>
    )
}
export default Header