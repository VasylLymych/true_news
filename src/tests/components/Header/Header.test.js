import Header from "../../../components/header/Header";
import {create} from "react-test-renderer"

describe("Header component test", () => {
    test("component snapshot",()=>{
        const component = create(<Header/>)
        expect(component).toMatchSnapshot()
    })
})