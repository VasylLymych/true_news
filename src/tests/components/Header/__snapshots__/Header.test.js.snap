// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test component snapshot 1`] = `
<header
  className="header"
>
  <div
    className="logo_wrapper"
  >
    <img
      alt="logo"
      className="logo_img"
      src="logo.jpg"
    />
    <h1>
      Новини України та світу
    </h1>
  </div>
</header>
`;
